docs/configuration/prefill/eidas_company.rst

```diff
@@ -0,0 +1,45 @@
+.. _configuration_prefill_eidas_company:
+
+===================
+eIDAS for companies
+===================
```

Comment:
Is this a different prefill plugin than the one for natural persons? If not, shouldn't both flavours be documented in a single eidas plugin document instead of two?